src/_components/AxisX.svelte

```diff
@@ -54,7 +55,7 @@
 
 <g class='axis x-axis' class:snapTicks>
 	{#each tickVals as tick, i}
-		<g class='tick tick-{ i }' transform='translate({$xScale(tick)},{$yRange[0]})'>
+		<g class='tick tick-{ i }' transform='translate({$xScale(tick)},{max($yRange)})'>
```

Comment:
Could you do `Math.max(...$yRange)` instead to avoid the `d3-array` import?

Reply:
I can.  I only used `d3-array` since you already had it as a `devDependency` and it was a little cleaner (and what I used in LayerChart).

I can switch to `Math.max(...$yRange)` if you prefer.

Reply:
Yea may as well just to make it more vanilla where possible